src/main/java/at/hannibal2/skyhanni/features/bingo/card/BingoCardReader.kt

```diff
@@ -70,7 +91,7 @@ class BingoCardReader {
                 description = description.substring(2)
             }
 
-            val done = lore.any { it.contains("GOAL REACHED") }
+            val done = lore.any { goalReachedPattern.matches(it) }
```

Comment:
```suggestion
            val done = lore.any { goalReachedPattern.find(it) }
```
as this was a contains before